eks-log-collector.sh

```diff
@@ -116,63 +158,39 @@ is_diskfull()
 {
   try "check disk space usage"
 
-  threshold=70
-  i=2
-  result=`df -kh |grep -v "Filesystem" | awk '{ print $5 }' | sed 's/%//g'`
+  local threshold
+  local result
+  threshold=1500000
+  result=$(df / | grep -v "Filesystem" | awk '{ print $4 }')
```

Comment:
Can you add h flag to make it human readable instead of bytes?

Reply:
Later down, I tell the user the space needed:

```
$((threshold>>10))MB
```

As it stands, that is roughly 1500MB or 1.5GB.